lib/presentation/organisms/timer_organism.dart

```diff
@@ -100,29 +100,43 @@ class _TimerOrganismState extends State<TimerOrganism>
   }
 
   void secondaryButtonOnTap(BuildContext context) {
-    final Widget body = Column(
-      children: [
-        ToggleButtonMolecule(
-          text: 'Sound',
-          offIcon: Icons.music_off_rounded,
-          onIcon: Icons.music_note_rounded,
-          onChange: (bool value) =>
-              PlayerController.instance.setSilentState(!value),
-          initialValue: !PlayerController.instance.isSilent(),
-        ),
-        ToggleButtonMolecule(
-          text: 'Screen Lock',
-          offIcon: Icons.lock_clock,
-          onIcon: Icons.lock_open,
-          onChange: (bool value) {
-            lockScreen = value;
-            WakeLockController.instance.setWakeLock(lockScreen);
-          },
-          initialValue: lockScreen,
-        ),
-      ],
+    final Widget body = SizedBox(
+      width: 200,
```

Comment:
If we can set it with max width but not minimum like it behaved before that will be nice.